app/models/rental.rb

```diff
@@ -3,4 +3,10 @@ class Rental < ApplicationRecord
   
   belongs_to :customer
   belongs_to :video
+
+  # checkout method
+  def self.check_out(customer_id, video_id)
```

Comment:
Re: my comment above about this being a class method vs. instance method. Not sure if an instance method would work though, so maybe this is a bogus suggestion?

```suggestion
  def check_out
```

Reply:
An instance method may work, let's test it out during our zoom call~